src/commands/midi.ts

```diff
@@ -28,6 +28,7 @@ export function toggleSoundCues(): boolean {
 window.onDidChangeTextEditorSelection(playerContext);
```

Comment:
Unsure if this line actually does anything. Try removing it and seeing if it breaks anything. If it doesn't mess anything up, leave it removed.